### PR TITLE
Support SR-IOV Network operator for Ubuntu

### PR DESCRIPTION
**Pull request by Mmduh-483**
This patch add support for enabling RDMA and installing the needed packages on the Ubuntu OS and updating the Makefile rules for support deploying the operator on Ubuntu

### CODE REVIEW

File: bindata/scripts/enable-rdma.sh [hunk 1]
```diff
@@ -18,32 +23,16 @@ function trigger_udev_event {
...
-  echo "It's CoreOS, exit"
-  if kmod_isloaded; then
-    exit
-  else
-    exit 1
-  fi
-else
-  if [ -f /host/usr/libexec/rdma-init-kernel ]; then
-    echo "rdma.service installed"
+function check_rdma {
```
Comment (pliurh) [inline bindata/scripts/enable-rdma.sh:26]: 'check_rdma' is not so accurate. Maybe we shall change it to 'enable_rdma'

File: bindata/scripts/enable-rdma.sh [hunk 2]
```diff
@@ -52,4 +41,38 @@ else
...
+    fi
+  else
+    RDMA_CONDITION_FILE="/host/usr/libexec/rdma-init-kernel"
+    RDMA_SERVICE_NAME="rdma"
+    PACKAGE_MANAGER=yum
+
+    check_rdma
+  fi
+
+  check_rdma
```
Comment (pliurh) [inline bindata/scripts/enable-rdma.sh:67]: This line can be removed.

**Comment by Mmduh-483** [comment]
@zshi-redhat @pliurh any comments here?

**Comment by pliurh** [comment]
@Mmduh-483 We are in the Feature Freeze period of OCP 4.6. So the master branch is closed, non-bugfix PR cannot be merged. We can have it merged as soon as the master becomes open.

**Comment by Mmduh-483** [comment]
@zshi-redhat @adrianchiris The base is changed to `dev` branch due to the feature freeze

**Pull request merged**